.github/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
`blank_issues_enabled: false` disables the "open a blank issue" button.

We need that because we want GitHub to show our new issue templates vs the default blank issue.